saltstack/base/salt/telemetry/centos/init.sls

```diff
@@ -33,14 +33,14 @@ install_cdp_telemetry_rpm:
 install_lsb_core_for_fluent:
   cmd.run:
     - name: yum install -y redhat-lsb-core
-{% if use_telemetry_archive == "Yes" %}
-install_cdp_logging_rpm_from_archive:
-  cmd.run:
-   - name: "/cdp/telemetry/install-package.sh cdp_logging_agent {{ archive_base_url }} {{ archive_credentials }} {{ cdp_logging_agent_rpm_repo_url }}"
-{% elif cdp_logging_agent_rpm_repo_url %}
+{% if cdp_logging_agent_rpm_url %}
 install_cdp_logging_rpm_from_repo_url:
   cmd.run:
-    - name: rpm -i {{ cdp_logging_agent_rpm_repo_url }}
+    - name: rpm -i {{ cdp_logging_agent_rpm_url }}
+{% elif use_telemetry_archive == "Yes" %}
+install_cdp_logging_rpm_from_archive:
+  cmd.run:
+   - name: "/cdp/telemetry/install-package.sh cdp_logging_agent {{ archive_base_url }} {{ archive_credentials }} {{ cdp_logging_agent_rpm_url }}"
```

Comment:
cdp_logging_agent_rpm_url will be empty here beause of the if confition in the line 39 . Do I miss something?

Reply:
Yes, that's a harmless side effect of switching up the "if"s.

Since I plan to open a separate ticket for cleaning up this mess anyway, I left it that way.